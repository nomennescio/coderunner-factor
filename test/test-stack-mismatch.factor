! Copyright 2019 nomennescio

USING: tools.testest ;
IN: tests

: run-tests ( -- )

"group of tests" describe#{
  "single test" it#{
    <{ 0 0 -> 0 }>
  }#
  "double test" it#{
    <{ 0 0 -> 0 }>
    <{ 0 -> 0 }>
  }#
  "double fail test" it#{
    <{ 0 0 -> 0 }>
    <{ 0 -> 0 0 0 }>
  }#
}#

;

MAIN: run-tests

