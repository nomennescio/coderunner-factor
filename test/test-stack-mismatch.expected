
<DESCRIBE::>group of tests

<IT::>single test

<FAILED::>Test Failed : Expected :<:LF:>0<:LF:>but got :<:LF:>0<:LF:>0

<COMPLETEDIN::>0.126526 ms

<IT::>double test

<FAILED::>Test Failed : Expected :<:LF:>0<:LF:>but got :<:LF:>0<:LF:>0

<PASSED::>Test Passed

<COMPLETEDIN::>0.032765 ms

<IT::>double fail test

<FAILED::>Test Failed : Expected :<:LF:>0<:LF:>but got :<:LF:>0<:LF:>0

<FAILED::>Test Failed : Expected :<:LF:>0<:LF:>0<:LF:>0<:LF:>but got :<:LF:>0

<COMPLETEDIN::>0.030305 ms

<COMPLETEDIN::>0.396158 ms
